src/components/Resume/Stars.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import starsfilled from '../../icons/star-fill.svg'
+import starsblank from '../../icons/star-blank.svg'
+
+const Stars = ({ lev }) => {
+    const stars = [];
+    for (let i = 1; i <= level(lev); i++) {
+        stars.push(<img src={starsfilled} key={i} alt="filled-star" />)
+    }
+    for (let i = level(lev) + 1; i <= 5; i++) {
+        stars.push(<img src={starsblank} key={i} alt="blank-star" />)
+    }
+    return (
+        <div className="keyword-level">
+            {stars}
+        </div>
+    );
+}
+
```

Comment:
Missing propTypes for Starts.

Reply:
Yes, `propTypes` are included to add a small layer of type safety when passing props. Additionally, we can enforce certain props to be required. Here is the documentation for how to use: https://reactjs.org/docs/typechecking-with-proptypes.html

Let me know if you need further help with this.

Reply:
> Actually I haven't use propTypes before. Can you please help me in that.

Yes, propTypes are included to add a small layer of type safety when passing props. Additionally, we can enforce certain props to be required. Here is the documentation for how to use: https://reactjs.org/docs/typechecking-with-proptypes.html

Let me know if you need further help with this.